step_definitions/createTeam-steps.js

```diff
@@ -10,6 +10,9 @@ const elementHelpers = require('../runtime/helpers/elementHelpers.js');
 let fullname1;
 let fullname2;
 let teamName;
+const student1 = 'Marla Mathe';
+const student2 = 'Waldemar Wunderlich';
+const teacher = 'lehrer@schul-cloud.org';
```

Comment:
move "const teacher = 'lehrer@schul-cloud.org'" to loginData.js

Reply:
done

Reply:
the both students as well